Sources/Nimble/Adapters/NimbleXCTestHandler.swift

```diff
@@ -43,11 +43,19 @@ class NimbleXCTestUnavailableHandler: AssertionHandler {
     private(set) var currentTestCase: XCTestCase?
 
     @objc func testCaseWillStart(_ testCase: XCTestCase) {
+        #if swift(>=3.2)
+        _swift_reportFatalErrorsToDebugger = false
+        #endif
+
         currentTestCase = testCase
     }
 
     @objc func testCaseDidFinish(_ testCase: XCTestCase) {
         currentTestCase = nil
+
+        #if swift(>=3.2)
+        _swift_reportFatalErrorsToDebugger = true
```

Comment:
Would it make sense to stash the previous value so we can restore it, rather than assuming `true`?

Reply:
Addressed in 13f2cc8.